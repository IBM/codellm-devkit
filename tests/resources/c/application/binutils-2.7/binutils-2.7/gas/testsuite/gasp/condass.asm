	.AIF	1 EQ 1
	OK
	.AELSE
	BAD
	.AENDI
	.AIF	1 EQ 2
	BAD
	.AELSE
	OK
	.AENDI
	.AIF	1 EQ 2
	BAD
	.AELSE
	OK
	.AIF	1 EQ 2
	BAD
	.AELSE
	OK
	.AENDI
	.AENDI
	.AIF	1 LT  2
	OK
	.AENDI
	.AIF	1 EQ  2
	BAD
	.AENDI
	.AIF	1 NE  2
	OK
	.AENDI
	.AIF	1 LE  2
	OK
	.AENDI
	.AIF	1 GT  2
	BAD
	.AENDI
	.AIF	3 GE  2
	OK
	.AENDI
	.AIF	3 LT  2
	BAD
	.AENDI
	.AIF	3 EQ  2
	BAD 
	.AENDI
	.AIF	3 NE  2
	OK
	.AENDI
	.AIF	3 LE  2
	BAD
	.AENDI
	.AIF	3 GT  2
	OK
	.AENDI
	.AIF	3 GE  2
	OK
	.AENDI
	.AIF	"FOO" EQ "BAR"
	BAD
	.AENDI
	.AIF	"FOO" EQ "FOO"
	OK
	.AENDI
	.AIF	"FOO" NE "BAR"
	OK
	.AENDI
	.AIF	"FOO" EQ "FOO"
	OK
	.AENDI
	.AIF	1 EQ 1
	.AIF	1 EQ 1
	OK
	.AELSE	
	BAD
	.AENDI
	.AIF	1 EQ 0
	BAD
	.AELSE	
	OK
	.AENDI
	OK
	.AELSE	
	BAD
	.AENDI
	.AIF	1 EQ 0
	BAD
	.AELSE	
	OK
	.AENDI
	.AIF	1 EQ 1
	OK
	.AELSE	
	BAD
	.AENDI
	.AIF	1 EQ 0
	BAD
	.AELSE	
	.AIF	1 EQ 1
	OK
	.AELSE	
	BAD
	.AENDI
	.AIF	1 EQ 0
	BAD
	.AELSE	
	OK
	.AENDI
	OK
	.AENDI
	.AIF	1 EQ 1
	OK
	.AIF	1 EQ 1
	OK
	.AELSE	
	BAD
	.AENDI
	.AIF	1 EQ 0
	BAD
	.AELSE	
	OK
	.AENDI
	.AELSE	
	BAD
	.AENDI
	.AIF	1 EQ 0
	BAD
	.AELSE	
	OK
	.AENDI
	.END
