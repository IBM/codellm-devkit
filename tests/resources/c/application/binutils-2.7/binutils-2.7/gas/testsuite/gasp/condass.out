!	.AIF	1 EQ 1
!	OK
	OK
!	.AELSE
!	BAD
!	.AENDI
!	.AIF	1 EQ 2
!	BAD
!	.AELSE
!	OK
	OK
!	.AENDI
!	.AIF	1 EQ 2
!	BAD
!	.AELSE
!	OK
	OK
!	.AIF	1 EQ 2
!	BAD
!	.AELSE
!	OK
	OK
!	.AENDI
!	.AENDI
!	.AIF	1 LT  2
!	OK
	OK
!	.AENDI
!	.AIF	1 EQ  2
!	BAD
!	.AENDI
!	.AIF	1 NE  2
!	OK
	OK
!	.AENDI
!	.AIF	1 LE  2
!	OK
	OK
!	.AENDI
!	.AIF	1 GT  2
!	BAD
!	.AENDI
!	.AIF	3 GE  2
!	OK
	OK
!	.AENDI
!	.AIF	3 LT  2
!	BAD
!	.AENDI
!	.AIF	3 EQ  2
!	BAD 
!	.AENDI
!	.AIF	3 NE  2
!	OK
	OK
!	.AENDI
!	.AIF	3 LE  2
!	BAD
!	.AENDI
!	.AIF	3 GT  2
!	OK
	OK
!	.AENDI
!	.AIF	3 GE  2
!	OK
	OK
!	.AENDI
!	.AIF	"FOO" EQ "BAR"
!	BAD
!	.AENDI
!	.AIF	"FOO" EQ "FOO"
!	OK
	OK
!	.AENDI
!	.AIF	"FOO" NE "BAR"
!	OK
	OK
!	.AENDI
!	.AIF	"FOO" EQ "FOO"
!	OK
	OK
!	.AENDI
!	.AIF	1 EQ 1
!	.AIF	1 EQ 1
!	OK
	OK
!	.AELSE	
!	BAD
!	.AENDI
!	.AIF	1 EQ 0
!	BAD
!	.AELSE	
!	OK
	OK
!	.AENDI
!	OK
	OK
!	.AELSE	
!	BAD
!	.AENDI
!	.AIF	1 EQ 0
!	BAD
!	.AELSE	
!	OK
	OK
!	.AENDI
!	.AIF	1 EQ 1
!	OK
	OK
!	.AELSE	
!	BAD
!	.AENDI
!	.AIF	1 EQ 0
!	BAD
!	.AELSE	
!	.AIF	1 EQ 1
!	OK
	OK
!	.AELSE	
!	BAD
!	.AENDI
!	.AIF	1 EQ 0
!	BAD
!	.AELSE	
!	OK
	OK
!	.AENDI
!	OK
	OK
!	.AENDI
!	.AIF	1 EQ 1
!	OK
	OK
!	.AIF	1 EQ 1
!	OK
	OK
!	.AELSE	
!	BAD
!	.AENDI
!	.AIF	1 EQ 0
!	BAD
!	.AELSE	
!	OK
	OK
!	.AENDI
!	.AELSE	
!	BAD
!	.AENDI
!	.AIF	1 EQ 0
!	BAD
!	.AELSE	
!	OK
	OK
!	.AENDI
!	.END
